torch/testing/_internal/common_methods_invocations.py

```diff
@@ -240,6 +287,52 @@ def sample_inputs(self, device, dtype, requires_grad=False):
                        SkipInfo('TestUnaryUfuncs', 'test_reference_numerics', device_type='cpu',
                                 dtypes=[torch.cfloat, torch.cdouble], active_if=IS_MACOS),
                    )),
+    UnaryUfuncInfo('log',
+                   ref=np.log,
+                   domain=(0, float('inf')),
+                   skips=(
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics',
+                                device_type='cpu', dtypes=[torch.bfloat16]),
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics',
+                                device_type='cuda', dtypes=[torch.cfloat, torch.cdouble]),
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics',
+                                device_type='cpu', dtypes=[torch.cfloat, torch.cdouble],
+                                active_if=IS_WINDOWS),
+                   )),
+    UnaryUfuncInfo('log10',
+                   ref=np.log10,
+                   domain=(0, float('inf')),
+                   decorators=(precisionOverride({torch.bfloat16: 1e-2}),),
+                   skips=(
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics',
```

Comment:
Eventually need to re-enable some comparison vs. reference on "test_easy_reference_numerics" or using golden values-based testing.

Reply:
Next PR: "easy_reference_numerics" where only values from -10 to 10 (intersected with domain) are tested

Reply:
Duplicate test_reference_numerics for easy (small, in domain?) values vs. hard (large, + outside of domain).